gazebo/rendering/Light.cc

```diff
@@ -584,13 +585,19 @@ void Light::SetRange(const double _range)
 //////////////////////////////////////////////////
 void Light::SetCastShadows(const bool _cast)
 {
-  if (this->dataPtr->light->getType() == Ogre::Light::LT_DIRECTIONAL)
-  {
-    this->dataPtr->light->setCastShadows(_cast);
-  }
-  else
+  this->dataPtr->light->setCastShadows(_cast);
```

Comment:
should we call this for point lights?

Reply:
I think code code should make point light generate shadow maps but because I didn't write any shaders and demo world to verify this, I decided to disable point light shadows for now. 7aa1d35.